src/vs/workbench/contrib/tags/electron-browser/workspaceTagsService.ts

```diff
@@ -447,7 +451,7 @@ export class WorkspaceTagsService implements IWorkspaceTagsService {
 			const packageJsonPromises = getFilePromises('package.json', this.fileService, this.textFileService, content => {
 				try {
 					const packageJsonContents = JSON.parse(content.value);
-					let dependencies = Object.keys(packageJsonContents['dependencies']).concat(Object.keys(packageJsonContents['devDependencies']));
+					let dependencies = Object.keys(packageJsonContents['dependencies'] || {}).concat(Object.keys(packageJsonContents['devDependencies'] || {}));
```

Comment:
@pierceboggan made a PR that fixed this, so could you edit your PR to just add the tags?

Reply:
Done!